internal/pkg/service/stream/keboolasink/bridge/fixtures/create_sink_snapshot.txt

```diff
@@ -148,7 +148,8 @@ storage/slice/level/local/123/456/my-source/my-sink/2000-01-01T01:00:00.000Z/my-
   },
   "local": {
     "dir": "123/456/my-source/my-sink/2000-01-01T01-00-00-000Z/2000-01-01T01-00-00-000Z",
-    "filename": "slice.csv.gz",
+    "filenamePrefix": "slice",
+    "filenameExtension": "csv.gz",
     "allocatedDiskSpace": "100MB"
```

Comment:
I need to split the `filename` field to the two, will be explained later (writing from each source node requires a separate disk file).